src/test/java/uk/gov/ons/ctp/response/action/service/ActionServiceTest.java

```diff
@@ -211,6 +212,17 @@ public void testUpdateActionNoUpdate() {
     assertThat(existingAction).isEqualTo(actions.get(3));
   }
 
+  @Test
+  public void testRerunAction() throws CTPException {
+    final Action action = actions.get(0);
+    when(actionRepo.findById(ACTION_ID_0)).thenReturn(action);
+    when(actionRepo.saveAndFlush(any())).then(returnsFirstArg());
```

Comment:
You don't need this when since we're not using the return value when we save the action